birdhouse/env.local.example

```diff
@@ -398,6 +398,27 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 # Magpie DB name
 #export MAGPIE_DB_NAME="magpiedb"
 
+# Magpie user registration and approvals
+#export MAGPIE_USER_REGISTRATION_ENABLED=false
+#export MAGPIE_USER_REGISTRATION_SUBMISSION_EMAIL_TEMPLATE=""
+#export MAGPIE_USER_REGISTRATION_APPROVAL_ENABLED=false
+#export MAGPIE_USER_REGISTRATION_APPROVAL_EMAIL_RECIPIENT=""
+#export MAGPIE_USER_REGISTRATION_APPROVAL_EMAIL_TEMPLATE=""
+#export MAGPIE_USER_REGISTRATION_APPROVED_EMAIL_TEMPLATE=""
+#export MAGPIE_USER_REGISTRATION_DECLINED_EMAIL_TEMPLATE=""
+#export MAGPIE_USER_REGISTRATION_NOTIFY_ENABLED=false
+#export MAGPIE_USER_REGISTRATION_NOTIFY_EMAIL_RECIPIENT=""
+#export MAGPIE_USER_REGISTRATION_NOTIFY_EMAIL_TEMPLATE=""
+
+# Magpie user assignment to groups with terms & conditions
+#export MAGPIE_GROUP_TERMS_SUBMISSION_EMAIL_TEMPLATE=""
+#export MAGPIE_GROUP_TERMS_APPROVED_EMAIL_TEMPLATE=""
+
+# Magpie smtp server configuration
+#export MAGPIE_SMTP_FROM=""
+#export MAGPIE_SMTP_HOST=""
+#export MAGPIE_SMTP_PASSWORD=""
```

Comment:
Since false and empty string, I think they could be uncommented.

Reply:
I guess they could be uncommented, considering their values here, but I see the other empty values in this file are also commented. I think I will leave it as it is (commented) to be coherent with the rest.